kong/plugins/rate-limiting/handler.lua

```diff
@@ -12,6 +12,13 @@ local ngx_timer_at = ngx.timer.at
 
 local RATELIMIT_LIMIT = "X-RateLimit-Limit"
 local RATELIMIT_REMAINING = "X-RateLimit-Remaining"
+local RATELIMIT_RESET = "X-RateLimit-Reset"
+
+local SECONDS_IN_PERIOD = {
+  second = 1,
+  minute = 60,
+  hour = 3600
+}
```

Comment:
Maybe better to expose the [EXPIRATIONS constant](https://github.com/ioggstream/kong/blob/89afad90c6bb1d9ed2de666ceadeb438a543cadb/kong/plugins/rate-limiting/policies/init.lua#L54-L61) instead of duplicating it (it is also incomplete, see below comment).

Reply:
Sure! Will do!

Reply:
done in ff678caf5